base/system/winlogon/sas.c

```diff
@@ -367,13 +383,17 @@ PlayLogonSound(
 
 static
 VOID
-PlayLogoffSound(
-    _In_ PWLSESSION Session)
+PlayLogoffShutdownSound(
+    _In_ PWLSESSION Session,
+    _In_ BOOL bShutdown)
 {
     if (!ImpersonateLoggedOnUser(Session->UserToken))
         return;
 
-    PlaySoundRoutine(L"WindowsLogoff", FALSE, SND_ALIAS | SND_NODEFAULT);
+    /* NOTE: Logoff and shutdown sounds play synchronously */
+    PlaySoundRoutine(bShutdown ? L"SystemExit" : L"WindowsLogoff",
+                     FALSE,
+                     SND_ALIAS | SND_NODEFAULT);
```

Comment:
```suggestion
                     FALSE, SND_ALIAS | SND_NODEFAULT);
```

Reply:
I would like to use the current style because `bLogon` doesn't related to the flags and this follows the ReactOS coding style.